contracts/zaps/OlympusZapV2.sol

```diff
@@ -116,7 +116,7 @@ contract Olympus_V2_Zap_In is ZapBaseV3 {
         uint256 toInvest = _pullTokens(fromToken, amountIn, referral, true);
 
         // make sure "swapTarget" is approved to spend this contracts "fromToken"
-        _approveToken(fromToken, swapTarget, toInvest);
+        //_approveToken(fromToken, swapTarget, toInvest);
```

Comment:
You can remove this line instead of commenting it out.

Reply:
Comments removed